code-reviews/recipes/CSharp.md

```diff
@@ -6,15 +6,80 @@
 
 ## Linting
 
-[CSE](../../CSE.md) projects should check code style with the [StyleCop Analyzers for the .NET Compiler Platform](https://github.com/DotNetAnalyzers/StyleCopAnalyzers). The minimum rules set teams should adopt is the [Managed Recommended Rules](https://msdn.microsoft.com/en-us/library/dd264893.aspx) rule set.
+We strongly believe that consistent style increases readability and maintainability of a code base. Hence we are recommending analyzers / linters to enforce consistency and style rules.
 
-Reference: [Setting up code analysis for a project](https://docs.microsoft.com/en-us/visualstudio/code-quality/code-analysis-for-managed-code-overview).
+### Project Setup
+
+We recommend using a common setup for your solution that you can refer to in all the projects that are part of the solution. Create a common.props file that contains the defaults for all of your projects:
+
+```xml
```

Comment:
Wonder if you want to set warning as errors to make the build fail if any of the warning is raised.
```xml
<PropertyGroup>
    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
    <WarningsAsErrors />
</PropertyGroup>
```

Reply:
You can configure what you want to be an error and what to be a warning. Individual configurations can be done on a per project basis. We can include it for completeness. It's also more conservative and could be used as a start.